package.json

```diff
@@ -12,7 +12,7 @@
     "clean": "rm -rf ./dist",
     "typecheck": "tsc --noEmit && tsc -p test/tsconfig.json --noEmit",
     "prepublishOnly": "pnpm clean && pnpm build && chmod +x ./dist/bin/cli.js && pnpm test",
-    "build": "tsx ./src/bin/index.ts --runtime node",
+    "build": "SWC_NODE_IGNORE_DYNAMIC=true node -r @swc-node/register ./src/bin/index.ts --runtime node",
```

Comment:
`SWC_NODE_IGNORE_DYNAMIC` is introduced in https://github.com/swc-project/swc-node/pull/745.